classes/controller/menu/class-postmeta-check-controller.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Post meta duplicate records check class
+ *
+ * @package P4BKS\Controllers\Menu
+ */
+
+namespace P4GBKS\Controllers\Menu;
+
+use P4GBKS\Command\Remove_Duplicate_Postmeta;
+
+if ( ! class_exists( 'Postmeta_Check_Controller' ) ) {
+
+	/**
+	 * Class Postmeta_Check_Controller
+	 */
+	class Postmeta_Check_Controller extends Controller {
+		/**
+		 * Create menu/submenu entry.
+		 */
+		public function create_admin_menu() {
+			if ( ! current_user_can( 'manage_options' ) ) {
+				return;
+			}
+
+			$current_user = wp_get_current_user();
+
+			if ( in_array( 'administrator', $current_user->roles, true ) && current_user_can( 'manage_options' ) ) {
```

Comment:
`current_user_can( 'manage_options')` will always be true at this point, as the function returns earlier if it's not.